packages/flutter/lib/src/material/stepper.dart

```diff
@@ -774,9 +804,10 @@ class _StepperState extends State<Stepper> with TickerProviderStateMixin {
         if (!_isLast(i))
           Expanded(
             child: Container(
+              key: Key('line$i'),
               margin: const EdgeInsets.symmetric(horizontal: 8.0),
-              height: 1.0,
-              color: Colors.grey.shade400,
+              height: widget.connectorThickness,
+              color: _connectorColor(widget.steps[i+1].isActive),
```

Comment:
According to the material1 guideline usage doc, it should be exactly 1.0 in thickness. Here is the [usage doc](https://m1.material.io/components/steppers.html#steppers-usage).